Some improvements on the 2.6 migration guide (#27094)
akka/akka: akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

the default dispatcher has been adjusted down to `1.0` which means the number of threads will be one per core, but at least
`8` and at most `64`. This can be tuned using the individual settings in `akka.actor.default-dispatcher.fork-join-executor`.

## Akka now uses Fork Join Pool from JDK

> Previously, Akka contained a shaded copy of the ForkJoinPool. In benchmarks, we could not find significant benefits of keeping our own copy, so from Akka 2.6 on, the default FJP from the JDK will be used. The Akka FJP copy was removed.

Reviewer: ```suggestion
Previously, Akka contained a shaded copy of the ForkJoinPool. In benchmarks, significant benefits of keeping our own copy were not found. From Akka 2.6 on, the default FJP is from the JDK and the Akka FJP copy is removed.
```

for how to do this.

> ### Migration from 2.5.x Artery to 2.6.x Artery

Reviewer 2: note that there were some structural changes in https://github.com/akka/akka/pull/27025
is this still correct?

* `akka.remote.artery.advanced.connection-timeout` to `akka.remote.artery.advanced.tcp.connection-timeout`

### Remaining with Classic remoting (not recommended)

> Classic remoting is deprecated but can be used in `2.6.` First you need to explicitly disable Artery by setting property `akka.remote.artery.enabled` to `false`. Further, any configuration under `akka.remote` that is

Reviewer: ```suggestion
Classic remoting is deprecated but can be used in `2.6.` Explicitly disable Artery by setting property `akka.remote.artery.enabled` to `false`. Further, any configuration under `akka.remote` that is
```

> Classic remoting is deprecated but can be used in `2.6.` First you need to explicitly disable Artery by setting property `akka.remote.artery.enabled` to `false`. Further, any configuration under `akka.remote` that is

Reviewer 2: good catch that the most essential part was missing
